scripts/build_platform_config.py

```diff
@@ -266,7 +268,7 @@ def codeOutDevicePin(device, pin, definition_name):
   #codeOut("#define JSVAR_CACHE_SIZE                "+str(200)+" // Number of JavaScript variables in RAM")
 else:
   codeOut("#define JSVAR_CACHE_SIZE                "+str(variables)+" // Number of JavaScript variables in RAM")
-  codeOut("#define FLASH_AVAILABLE_FOR_CODE        "+str(flash_available_for_code))
+  codeOut("#define FLASH_AVAILABLE_FOR_CODE        "+str(int(flash_available_for_code)))
```

Comment:
Oh great. God knows why everyone likes Python so much :(

I just tested and on Python 2, `2/1 == 1`, but on Python 2 it's `1.0`.  All those things should be integers - so I guess they should all be cast to ints.

What system are you running the build on such that the default is Python 3? I'm using Ubuntu here and while I have 3 installed, the default is still 2.